### ISSUE TITLE
PhpdocVarWithoutNameFixer not removing var name in single line blocks

### ISSUE THREAD

**Issue by kubawerlos** [FriendsOfPHP/PHP-CS-Fixer#3962]
Why is [that](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.12.2/src/Fixer/Phpdoc/PhpdocVarWithoutNameFixer.php#L73)? Seems like a bug to me. In
```php
<?php
class Foo {
    /** @var Bar $bar */
    private $bar;
}
```
the variable name should be removed.

**Comment by dmvdbrugge** [comment]
Duplicate of #3112, fixed in #3911

**Comment by kubawerlos** [comment]
OMG, sorry.
